openapi/vonage.numberinsight/types.bal

```diff
@@ -240,17 +238,23 @@ public type NiResponseJsonAdvanced record {
     # If the user has changed carrier for `number`. The assumed status means that the information supplier has replied to the request but has not said explicitly that the number is ported.
     string? ported?;
     # Information about the roaming status for `number`. This is applicable to mobile numbers only.
-    NiRoaming|string? roaming?;
+    NiRoaming roaming?;
     # Information about the network `number` is currently connected to.
     NiCallerIdentity caller_identity?;
     # Shows if all information about a phone number has been returned. Possible values:
+    # 
+    # Code | Text
+    # --- | ---
+    # 0 | Success
+    # 1 | Partial success - some fields populated
+    # 2 | Failed
     int lookup_outcome?;
     # Shows if all information about a phone number has been returned.
     string lookup_outcome_message?;
     # Does `number` exist. `unknown` means the number could not be validated. `valid` means the number is valid. `not_valid` means the number is not valid. `inferred_not_valid` means that the number could not be determined as valid or invalid via an external system and the best guess is that the number is invalid. This is applicable to mobile numbers only.
     string valid_number?;
     # Can you call `number` now. This is applicable to mobile numbers only.
-    string? reachable?;
+    string reachable?;
```

Comment:
@RolandHewage could u remember the reason to manually update `reachable` to a nillable field in the initial commit? Not defined as `nullable: true` in the OpenAPI file. 
```
        reachable:
          type: string
          description: "Can you call `number` now. This is applicable to mobile numbers only."
          enum:
            - unknown
            - reachable
            - undeliverable
            - absent
            - bad_number
            - blacklisted
          example: "reachable"
```